_P176_CDM7160.ino

```diff
@@ -0,0 +1,321 @@
+//#######################################################################################################
+//######################## Plugin 176 CDM7160 I2C CO2 Sensor ############################################
+//#######################################################################################################
+// development version
+// by: V0JT4
+
+
+#define PLUGIN_176
+#define PLUGIN_ID_176        176
+#define PLUGIN_NAME_176       "Gases - CO2 CDM7160"
+#define PLUGIN_VALUENAME1_176 "CO2"
+#define PLUGIN_VALUENAME2_176 "BUSY"
+
+boolean Plugin_176_init = false;
+
+
+#define CDM7160_ADDR   0x69 // default address
+#define CDM7160_ADDR_0 0x68 // CAD0 tied to GND
+
+#define  CDM7160_REG_RESET   0x00
+#define  CDM7160_REG_CTL     0x01
+#define  CDM7160_REG_STATUS  0x02
+#define  CDM7160_REG_DATA    0x03
+#define  CDM7160_REG_HIT     0x0A
+#define  CDM7160_REG_FUNC    0x0F
+
+#define  CDM7160_FLAG_REST  0x01
+#define  CDM7160_FLAG_DOWN  0x00
+#define  CDM7160_FLAG_CONT  0x06
+#define  CDM7160_FLAG_BUSY  0x80
+#define  CDM7160_FLAG_HPAE  0x04
+#define  CDM7160_FLAG_PWME  0x01
+
+
+byte plugin_176_i2caddr;
+
+boolean plugin_176_begin()
+{
+  Wire.begin();
+  return(true);
+}
+
+boolean plugin_176_readByte(unsigned char address, unsigned char &value)
```

Comment:
Why are you  not using the functions in `I2C.ino` ?

Reply:
Mainly because I based it on code from _P015_TSL2561.ino but I'll change it to functions in I2C.ino and use them in planned CM1107 plugin if Cubic sends me one finally.

Reply:
> if Cubic sends me one finally.

Is that something you need my help for? And if so, what do you need?